performanceplatform/collector/ga/contrib/content/table.py

```diff
@@ -52,3 +44,19 @@ def gen():
         documents.extend(ds)
 
     Pusher(data_set_config, options).push(documents)
+
+
+def get_department(filters):
+    for f in filters:
+        try:
+            filter_key, filter_value = re.split('=~\^', f)
+            return try_get_department(filter_value)
+        except:
+            raise ValueError("department not found in filters, expected "
+                             "filter expression expression containing "
+                             "'=~^' e.g. 'Organisation=~^<A1>'.")
+    else:
+        # If no matches, raise a RuntimeError
+        raise RuntimeError("department not found in filters, expected "
```

Comment:
@leenagupte: Hey. I just mean that in the case where there is nothing in the filters (the other paths either returning from try_get_department or raising a ValueError) the the RuntimeError will get raised with a message claiming the expression needs to begin with "'customVarValue9=~^'." but isn't this now out of data and misleading as there are other ways to construct filters?